eos-knowledge-lib.modules

```diff
@@ -241,6 +241,13 @@ use the latest stable release branch from GNOME if you prefer that.
     </dependencies>
   </autotools>
 
+  <autotools id="emeus">
+    <branch repo="github" module="ebassi/emeus"/>
+    <dependencies>
+      <dep package="gtk+"/>
```

Comment:
Both meson and ninja should be part of the development tools needed to use jhbuild. The modules file does not specify a dependency on autoconf/automake/make either.